AcadeMeData/migrations/0009_Course_test_data.py

```diff
@@ -0,0 +1,39 @@
+from django.db import migrations, transaction
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '__latest__'),
```

Comment:
Is it needed? do you have user dependency for those tests?
Maybe University? and Professor?

('AcadeMeData', '0008_Professor_test_data'),

Reply:
I don't really understand what this line does but without it the migrations don't work..
I have dependencies on 'Professor' and 'Degree' models.

Reply:
maybe you should do google  django migration dependencies ?
I found this, https://docs.djangoproject.com/en/3.2/howto/writing-migrations/

Reply:
Line removed. :)